### DISCUSSION TITLE
What is the etiquette on commenting on scouted links

### DISCUSSION THREAD

**Discussion by closetgeekshow** [dailydotdev/daily#895] [General]
I just today got the ability to submit links, which is great because I find a couple things per day that I want to post. What I'm unsure about is commenting once it's posted. 

Can I post a comment immediately or should I wait for others to engage a discussion first? On the one hand, I generally have a thought to share, that's why I want to post it. On the other I feel like this might be considered bad form for some reason I can't quite put my finger on.

**Comment by idoshamun** [comment]
Hi @closetgeekshow,

That's a solid point that we still need to address. Feel free to drop a comment right away. 
It can be helpful for other community members.
Feel free to drop any other questions in this regard.
